docs/appendices/release-notes/unreleased.rst

```diff
@@ -61,6 +61,8 @@ Changes
 - Optimized the evaluation of ``CASE`` expressions to prevent stack overflows
   for very large expressions.
 
+- Allowed schema and table names to contain upper case letters.
```

Comment:
Please add another phrase, that this is allowed when quoted and please adapt the docs in https://github.com/crate/crate/blob/master/docs/general/ddl/create-table.rst & https://github.com/crate/crate/blob/master/docs/sql/general/lexical-structure.rst#id9

Reply:
Thank you for catching this.